core/src/main/kotlin/Utilities/DownloadSamples.kt

```diff
@@ -0,0 +1,64 @@
+package org.jetbrains.dokka.Utilities
```

Comment:
Add a copyright header (grab any of the Google ones from any of the other files)